image/templates/helm/stackrox-central/templates/99-openshift-monitoring.yaml

```diff
@@ -49,7 +49,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: central-monitor
```

Comment:
What happens if we install another Central chart? We should make sure that the name is unique, such that there are no collisions between different Helm releases / CRs.

Reply:
I'll add monitored namespace name to prevent it.

Reply:
I'm not sure that's enough, because there can we multiple Central CRs / Helm releases in the same namespace. Perhaps a combination of monitored namespace and release name?